dataactbroker/handlers/fileHandler.py

```diff
@@ -969,7 +969,7 @@ def build_file_map(self, request_params, file_type_list, response_dict, upload_f
             if file_name:
                 if not self.isLocal:
                     upload_name = "{}/{}".format(
-                        g.user.user_id,
+                        submission.submission_id,
```

Comment:
Yeah the latter was my concern, when a user updates a submission, this'll put the new file in the right place. Looks good to me 👍